pkg/products/solutionexplorer/reconciler.go

```diff
@@ -57,6 +57,7 @@ const (
 	defaultRouteName          = "solution-explorer"
 	manifestPackage           = "integreatly-solution-explorer"
 	paramRoutingSubdomain     = "ROUTING_SUBDOMAIN"
+	paramInstallationType = "INSTALLATION TYPE"
```

Comment:
Missing an underscore
```suggestion
	paramInstallationType = "INSTALLATION_TYPE"
```